### ISSUE TITLE
Feature request: Display conferences on a map

### ISSUE THREAD

**Issue by benjaminoakes** [ruby-conferences/ruby-conferences.github.io#562]
It would be nice to have pins on a map indicating the location of each conference

**Comment by marcoroth** [comment]
That sounds like a cool idea! Do you want to take a stab at this @benjaminoakes?

**Comment by Morozzzko** [comment]
As a long-time user and subscriber for every updates, couldn't help but leave a comment

What would the use-case for a map be? See the actual GPS spot where the conference is to be located? Just highlight the country? Highlight the city?

While the broader navigation might be cool, it does seem like a pretty complicated thing to implement. A lot of upcoming conferences don't have an announced venue, some conferences change location, and contributors will need to track all of those things

**Comment by marcoroth** [comment]
Yeah, I agree. While it's a cool idea it's probably not as trivial as it seems. Unless we actually hard-code lat/lng coordinates for every event, which seems overkill.

We might be able to use the existing location text and just put the marker into the city, but I guess that doesn't really add a bunch of value.

**Comment by benjaminoakes** [comment]
Fair points.  My use case is that I'd like to easily see conferences near me. Perhaps there are alternative approaches to consider?

**Comment by Morozzzko** [comment]
I think it'd be nice to have a map with larger scope than just "specific lon/lat"

Like, there's gonna be a EuRuKo in Bosnia and Herzegovina. I live in Belgrade, Serbia, so the choice between Tuzla and Sarajevo means a lot to me – I'll either take a car or a plane. For most people, though, it's a question of "how far is this country/state from mine" so the finer details aren't that important

For folks in the US, there's probably a similar story – if a person lives in WA and there's a conference in OR, there's a chance they will care about the specific location. For most folks in say, CA, NY, TX and most other states – won't make much of a difference.

If that sounds like a valid use-case, then probably there's a room for map with country-level highlights & info. Perhaps, some kind of pin around the capital?

Although, it'll leave us with difficult choices because country borders are a mess. See [Terra nullus](https://en.wikipedia.org/wiki/Terra_nullius), for example. There are two cases when chunks of land are technically not claimed by anyone because both sides cannot agree on the specific boundary between the two countries. And we're not even touching partially recognized states. So, to draw a line between the countries, is to take a stand in quite a few political topics. 
We can probably outsource this decision to vendors like OpenStreetMaps, though

**Comment by marcoroth** [comment]
It's not quite a map, but I guess it's a start: https://github.com/ruby-conferences/ruby-conferences.github.io/pull/748
